projects/ccf-eui/src/app/app.component.html

```diff
@@ -37,6 +37,7 @@
       <ccf-results-browser
         [dataLoading]="spinnerActive$ | async"
         [data]="data.listData$ | async"
```

Comment:
@adphillips2017 Is this still needed?

Reply:
No, but I thought you said to make it so that we could easily switch back to the old version?

Reply:
In your follow-up PR, you can remove the pieces that are no longer needed. I just wanted that stuff to stay until we had your stuff dropped in.